tests/playbooks/roles/install-k3s/defaults/main.yaml

```diff
@@ -8,4 +8,6 @@ sg_name: "k3s_sg"
 keypair_name: "k3s_keypair"
 image_url: "https://cloud-images.ubuntu.com/releases/focal/release/ubuntu-20.04-server-cloudimg-amd64.img"
 image_name: "ubuntu-focal"
-master_port_name: "k3s_master"
\ No newline at end of file
+master_port_name: "k3s_master"
+master_external_flag: "--disable-cloud-controller --kubelet-arg='cloud-provider=external'"
```

Comment:
I suggest using `k3s_master_cloud_provider_config` and `k3s_worker_cloud_provider_config`

Reply:
updated